vignettes/r2dii-analysis.Rmd

```diff
@@ -20,6 +20,12 @@ library(r2dii.match)
 library(r2dii.analysis)
 ```
 
+To plot your results, you may also load the package [r2dii.plot](https://2degreesinvesting.github.io/r2dii.plot).
+
+``` {r use-r2dii.plot}
+library(r2dii.plot)
+```
```

Comment:
Remember you need to install r2dii.plot on the CI workflow -- else website won't build:

https://github.com/2DegreesInvesting/r2dii.analysis/runs/3045513489

```
Error: Error in library(r2dii.plot) : there is no package called 'r2dii.plot'
```